apps/vmq_server/test/mqtt5_v4compat.erl

```diff
@@ -37,46 +40,63 @@ expect_packet_(4, Socket, Name, Packet) ->
 expect_packet_(5, Socket, _Name, ExpectedPacket) ->
     packetv5:expect_frame(Socket, ExpectedPacket).
 
+receive_at_most_n_frames(Socket, N, QoS, Config) ->
+    receive_at_most_n_frames_(protover(Config), Socket, N, QoS).
+
+receive_at_most_n_frames_(4, Socket, N, QoS) ->
+    lists:reverse(packet:receive_at_most_n_frames(Socket, N, QoS));
+receive_at_most_n_frames_(5, _Socket, _N, _QoS) ->
+    {error, not_implemented}.
+
 gen_connect(ClientId, Opts, Config) ->
     gen_connect_(protover(Config), ClientId, Opts).
 
 gen_connect_(4, ClientId, Opts) ->
-    packet:gen_connect(ClientId, [{protover, 4}|Opts]);
+    packet:gen_connect(ClientId, [{protover, 4} | Opts]);
 gen_connect_(5, ClientId, Opts) ->
     WillMsg = proplists:get_value(will_msg, Opts),
     WillTopic = proplists:get_value(will_topic, Opts),
     case {WillMsg, WillTopic} of
         {undefined, undefined} ->
             packetv5:gen_connect(ClientId, fixup_v4_opts(Opts));
-        {WillMsg, WillTopic} when WillMsg =/= undefined,
-                                  WillTopic =/= undefined ->
+        {WillMsg, WillTopic} when
```

Comment:
Not sure why the following formatting changed?

Reply:
Probably because the file saw addition of code and formatter decided to reformat other parts too.